frontend/public/components/masthead-toolbar.jsx

```diff
@@ -22,6 +23,25 @@ const UpdatesAvailableButton = ({obj, onClick}) => {
     : null;
 };
 
+const onServiceMesh = (url, name, e) => {
+  e.preventDefault();
+  window.open(`${url}?namespace=${name}`, '_blank').opener = null;
+};
+
+const ServiceMeshItem = ({namespace}) => {
+  if (namespace.loaded) {
+    const ns = namespace.data;
+    const serviceMeshUrl = _.get(ns, 'metadata.annotations["openshift.servicemesh/url"]');
+    const name = ns.metadata.name;
```

Comment:
I would inline this value. It's more obvious it's the name of the namespace that way, and we only use it once below.

Reply:
ok, changed to inline.